packages/client/src/client/App/Credit/CreditRfqForm/CreditInstrumentSearch.tsx

```diff
@@ -60,9 +60,9 @@ export const [useSelectedInstrument] = bind(
     withLatestFrom(creditInstrumentsByCusip$),
     map(([instrumentId, creditInstrumentsByCusip]) =>
       instrumentId !== null
-        ? Object.values(creditInstrumentsByCusip).find(
+        ? (Object.values(creditInstrumentsByCusip).find(
```

Comment:
only 1 mod in this project 😅 
surprising there was any change, tho, tbh as I only bumped 1 patch level!